aten/src/ATen/mps/MPSStream.mm

```diff
@@ -115,6 +118,13 @@ @interface MPSGraphExecutionDescriptor ()
   [_commandBuffer commitAndContinue];
 }
 
+void MPSStream::commitAndContinueRoot() {
+  assert(_commandBuffer);
+  id<MTLCommandBuffer> rootCommandBuffer = [_commandBuffer rootCommandBuffer];
+  [_commandBuffer commitAndContinue];
```

Comment:
I guess after this point we should check if the `rootCommandBuffer`'s status is `committed`. If so, then wait until completed.